src/common/check-usage.js

```diff
@@ -8,7 +8,9 @@ const checkUsage = function (manifest, varName) {
   for (let i = 0; i < containers.length; i++) {
     const container = containers[i]
     const envVars = container['environment']
-
+    if (!envVars) {
+      continue
+    }
     // Check if there is a key that corresponds to the variable name
     if (envVars['varName']) {
```

Comment:
I stumbled over this when I was working with this library myself. I think this is a bug. You meant:

```js
if (envVars[varName]) {
```